source/SkiaSharp.Views/SkiaSharp.Views.UWP/UWPExtensions.cs

```diff
@@ -2,7 +2,7 @@
 using Windows.Foundation;
 using Windows.UI;
 
-#if WINDOWS
+#if WINDOWS || WINUI
```

Comment:
It's likely that changing all references to WINUI would make more sense, which means adding `WINUI` as a constant in the existing WinUI project.